docs/cep/enrich-data/enrich-data-with-collection.md

```diff
@@ -9,9 +9,11 @@ For this purpose, consider a scenario where you receive sales records generated
 
 ## Create a Stream Worker
 
+Refer to the [Stream Workers](/cep/index.md) section for an explanation of how stream workers function.
```

Comment:
```suggestion
For more information about stream workers and how they work, refer [Stream Workers](index.md).
```

Reply:
Generally we want to tell them what to expect first, then give them the instruction and link to click.